### PR TITLE
Fix slash error with AzAuth

### PR DESCRIPTION
**Pull request by Riptiaz**
Use URL function to decode the URL

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
 {
-    "name": "minecraft-java-core",
-    "version": "3.10.4",
+    "name": "minecraft-java-core-azbetter",
```
Comment (GeekCornerGH) [inline package.json:2]: I don't think you'll want to add this in the PR
```suggestion
    "name": "minecraft-java-core",
```

File: package.json [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+    "version": "3.11.0",
```
Comment (GeekCornerGH) [inline package.json:3]: And I don't think you should be the one who bumps the deps
```suggestion
    "version": "3.10.4",
```

File: package.json [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
     "types": "./build/Index.d.ts",
     "exports": {
         ".": {
             "import": "./build/Index.js",
             "require": "./build/Index.js"
         }
     },
-    "description": "A library starting minecraft game NW.js and Electron.js",
+    "description": "A library starting minecraft game NW.js and Electron.js with better Azuriom support.",
```
Comment (GeekCornerGH) [inline package.json:11]: Again, you'll probably want to remove this part
```suggestion
    "description": "A library starting minecraft game NW.js and Electron.js",
```

File: package.json [hunk 4]
```diff
@@ -28,8 +28,8 @@
         "Forge",
         "Minecraft-Forge"
     ],
-    "author": "Luuxis",
-    "license": "CCANC",
+    "author": "Luuxis & Riptiaz",
+    "license": "SEE LICENSE IN LICENSE.md",
```
Comment (GeekCornerGH) [inline package.json:32]: Please revert this
```suggestion
    "author": "Luuxis",
    "license": "CCANC",
```

File: package.json [hunk 5]
```diff
@@ -47,11 +47,15 @@
         "typescript": "^4.9.4"
     },
     "bugs": {
-        "url": "https://github.com/luuxis/minecraft-java-core/issues"
+        "url": "https://github.com/riptiaz/minecraft-java-core/issues"
```
Comment (GeekCornerGH) [inline package.json:50]:
```suggestion
        "url": "https://github.com/luuxis/minecraft-java-core/issues"
```

File: package.json [hunk 6]
```diff
@@ -47,11 +47,15 @@
...
     },
-    "homepage": "https://github.com/luuxis/minecraft-java-core#readme",
+    "homepage": "https://github.com/riptiaz/minecraft-java-core#readme",
```
Comment (GeekCornerGH) [inline package.json:52]:
```suggestion
    "homepage": "https://github.com/luuxis/minecraft-java-core#readme",
```

File: package.json [hunk 7]
```diff
@@ -47,11 +47,15 @@
...
     "repository": {
         "type": "git",
-        "url": "git+https://github.com/luuxis/minecraft-java-core.git"
+        "url": "git+https://github.com/riptiaz/minecraft-java-core.git"
```
Comment (GeekCornerGH) [inline package.json:55]:
```suggestion
        "url": "git+https://github.com/luuxis/minecraft-java-core.git"
```

File: package.json [hunk 8]
```diff
@@ -47,11 +47,15 @@
...
+    },
+    "main": "index.js",
+    "directories": {
+        "test": "test"
```
Comment (GeekCornerGH) [inline package.json:59]: Why was this added?

**Review by GeekCornerGH — CHANGES_REQUESTED** [review]
Haven't tested yet, here are my first thoughts.

Also when making a PR on GitHub, you should consider creating your pull request from another branch than the default one

File: package.json (hunk 8 above)
Comment (Riptiaz) [reply package.json:59]: Idk, when I run npm init it was added, you can remove I think

**Comment by GeekCornerGH** [comment]
Fixed in https://github.com/luuxis/minecraft-java-core/commit/5fbb49d88f866fed9f308440d408677b026e8ba1

**Pull request closed without merging**